backend/server/common/annotations/local_file_csv.py

```diff
@@ -259,6 +260,7 @@ def update_parameters(self, parameters, data_adaptor):
         params = {}
         params["annotations"] = self.user_annotations_enabled()
         params["annotations_genesets_readonly"] = not self.gene_sets_save_enabled()
+        params["annotations_genesets_name_is_read_only"] = self.gene_sets_output_file is not None
```

Comment:
I might be misunderstanding how this is being used, but I think it should be         `params["annotations_genesets_name_is_read_only"] = self.gene_sets_output_file is None`
my interpretation is if there is no output file set it should be read only (param set to True) but if there is a value for the output file the param `annotations_genesets_name_is_read_only` should be false.

Reply:
This param indicates whether the annotation collection name is read-only (fixed by the backend) or read-write (the f/e will prompt the user for a name).  The current and only use for the param is to tell the f/e whether or not it should ask the user for a name.

When the backend already knows the file name, the name is read-only.  When the backend does not know a name, it sets this to True and the f/e prompts.  See:
* client/src/reducers/annotations.js: line 37
* client/src/components/autosave/filenameDialog.js: line 110

This is probably a case of a badly named parameter, predating this PR.

Reply:
ahh I figured I was missing something. Thanks!